odoo/service/server.py

```diff
@@ -97,9 +97,12 @@ def empty_pipe(fd):
 
 
 def cron_database_list():
-    if db_names := config['db_name']:
-        return db_names
-    return list_dbs(True)
+    db_names = list(config['db_name']) or list_dbs(True)
+    # When listing databases for cron jobs, return them in a random order.
+    # This avoids that all triggered workers start with the same database
+    # list in the same order.
+    random.shuffle(db_names)
```

Comment:
I don't think we want that.

Reply:
saas or single db instances are not impacted. It allows smaller multi-db instances to avoid having all workers trying to pick up the same db at the same time.
But not hard feelings if other people want to remove it.

Reply:
al: to remove